### PR TITLE
modules/SceIpmi: stub some functions.

### PR DESCRIPTION
**Pull request by Zangetsu38**
#About
- modules/SceIpmi: stub some functions.

Adding more function request for vsh.

### CODE REVIEW

File: vita3k/modules/module_parent.cpp [hunk 1]
```diff
@@ -129,6 +150,7 @@ void call_import(EmuEnvState &emuenv, CPUState &cpu, uint32_t nid, SceUID thread
 
             if (!emuenv.missing_nids.contains(nid) || LOG_UNK_NIDS_ALWAYS) {
                 LOG_ERROR("Import function for NID {} not found (thread name: {}, thread ID: {})", log_hex(nid), thread->name, thread_id);
+                LOG_DEBUG("{}\n{}", save_context(*thread->cpu).description(), thread->log_stack_traceback());
```
Comment (Macdu) [inline vita3k/modules/module_parent.cpp:153]: This is a traceback log I didn't remove.

File: vita3k/nids/include/nids/nids.inc [hunk 2]
```diff
@@ -2044,6 +2044,25 @@ NID(sceImeSetCaret, 0xD8342D2A)
...
+NID(_ZN4IPMI6Client17invokeAsyncMethodEjPKNS_8DataInfoEjPjPKNS0_12EventNotifeeE, 0x387AFA3F)
+NID(_ZN4IPMI6Client17invokeAsyncMethodEjPKvjPiPKNS0_12EventNotifeeE, 0xAFD10F3B)
+NID(_ZN4IPMI6Client19terminateConnectionEv, 0xD484D36D)
+NID(_ZN4IPMI6Client6Config24estimateClientMemorySizeEv, 0x4E255C31)
+NID(_ZN4IPMI6Client6createEPPS0_PKNS0_6ConfigEPvS6_, 0xB282B430)
+NID(_ZN4IPMI6Client6getMsgEjPvPjjS2_, 0xA3E650B0)
+NID(_ZN4IPMI6Client7connectEPKvjPi, 0xA22C3E01)
+NID(_ZN4IPMI6Client7destroyEv, 0x101C93F8)
+NID(_ZN4IPMI6Client9tryGetMsgEjPvPmm, 0x60EFADE7)
+NID(_ZN4IPMI6ClientD1Ev, 0xA5AA193C)
```
Comment (Macdu) [inline vita3k/nids/include/nids/nids.inc:2065]: It should be worth mentionning some of them are "fake" nids

File: vita3k/modules/module_parent.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/modules/module_parent.cpp:153]: Oh sorry lol oki

File: vita3k/nids/include/nids/nids.inc (hunk 2 above)
Comment (Zangetsu38) [reply vita3k/nids/include/nids/nids.inc:2065]: Nid is not sur ? 🤔

Comment (Zangetsu38) [reply vita3k/nids/include/nids/nids.inc:2065]: I have edited and added comment for nids

File: vita3k/modules/module_parent.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/modules/module_parent.cpp]: Done

**Review by Macdu — APPROVED** [review]

**Pull request merged**